js/packages/web/src/components/AuctionCard/index.tsx

```diff
@@ -618,6 +625,14 @@ export const AuctionCard = ({
                       );
                       setLastBid({ amount });
                       bidTxid = txid;
+                      track('instant_sale_completed', {
+                        category: 'auction',
+                        label: 'confirmed',
+                        ...value && {
+                          value: value * useSolPrice(), //Google Analytics likes this one in USD :)
+                          sol_value: value
+                        }
+                      })
```

Comment:
This action is actually the lister ending the auction early and reclaiming their NFT.

A good thing to track but should be instant_saled_canceled. And maybe track the bids submitted (that will be purchased) and total supply.